lib/matplotlib/projections/polar.py

```diff
@@ -591,11 +605,21 @@ def format_coord(self, theta, r):
         Return a format string formatting the coordinate using Unicode
         characters.
         """
+        # Strip away any units
+        r = self.convert_yunits(r)
+        if not isinstance( theta, float ):
+            theta = self.convert_xunits(theta)
+            # Stripping away units makes theta be in radians.
+            # convert to degrees
+            theta = theta * (180.0 / np.pi)
+
         theta /= math.pi
         # \u03b8: lower-case theta
         # \u03c0: lower-case pi
         # \u00b0: degree symbol
-        return '\u03b8=%0.3f\u03c0 (%0.3f\u00b0), r=%0.3f' % (theta, theta * 180.0, r)
+        s = '\u03b8=%0.3f\u03c0 (%0.3f\u00b0), r=%0.3f' % (theta, theta * 180.0, r)
+        s = s.encode( 'utf-8' )
```

Comment:
This shouldn't be necessary and probably indicates a bug somewhere else.  Returning unicode from this function is actually what we *want*.  Can you describe which backend/platform you were using when you can across this problem?